src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+
+}
+
+.logo__img {
+  padding: 10px 0;
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  display: block;
+
+}
```

Comment:
1. Move font related styles to `html` or `:root` selector, cause it is a good practice and those styles are actually 'global'
2. `display: block;` is redundant